JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv

```diff
@@ -83,17 +53,7 @@ receivesRadiation,Object Property,,,https://w3id.org/bot#Building UNION SolarDev
 PVTPlateCollectorsArea,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,IS-A,,,,The total area of photovoltaic thermal collectors with thermal plate collectors,https://www.theworldavatar.com/kg/ontoubemmp/,
-RoofPVTPlateCollectors,Class,PVTPlateCollectors,IS-A,,,,PVT Plate Collectors with roof type,https://www.theworldavatar.com/kg/ontoubemmp/,
-WallPVTPlateCollectors,Class,PVTPlateCollectors,IS-A,,,,PVT Plate Collectors with wall type,https://www.theworldavatar.com/kg/ontoubemmp/,
-EastWallPVTPlateCollectors,Class,WallPVTPlateCollectors,IS-A,,,,Wall PVT Plate Collectors with east orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-WestWallPVTPlateCollectors,Class,WallPVTPlateCollectors,IS-A,,,,Wall PVT Plate Collectors with west orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-NorthWallPVTPlateCollectors,Class,WallPVTPlateCollectors,IS-A,,,,Wall PVT Plate Collectors with north orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-SouthWallPVTPlateCollectors,Class,WallPVTPlateCollectors,IS-A,,,,Wall PVT Plate Collectors with south orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
 PVTTubeCollectors,Class,PVTCollectors,IS-A,,,,Photovoltaic thermal hybrid solar collector with thermal tube collector,https://www.theworldavatar.com/kg/ontoubemmp/,
 PVTTubeCollectorsArea,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,IS-A,,,,The total area of photovoltaic thermal collectors with thermal tube collectors,https://www.theworldavatar.com/kg/ontoubemmp/,
-RoofPVTTubeCollectors,Class,PVTTubeCollectors,IS-A,,,,PVT Tube Collectors with roof type,https://www.theworldavatar.com/kg/ontoubemmp/,
-WallPVTTubeCollectors,Class,PVTTubeCollectors,IS-A,,,,PVT Tube Collectors with wall type,https://www.theworldavatar.com/kg/ontoubemmp/,
-EastWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with east orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-WestWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with west orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-NorthWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with north orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-SouthWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with south orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
+hasSolarSuitableArea,Object Property,,,https://www.theworldavatar.com/kg/ontobuildingstructure/RoofFacade UNION https://www.theworldavatar.com/kg/ontobuildingstructure/WallFacade,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,A relationship linking the building's roof or wall fa�ade with their area that is suitable for receiving solar energy.,https://www.theworldavatar.com/kg/ontoubemmp/,
```

Comment:
Please fix the dodgy character in this and the subsequent line (and regenerate the owl). You'll need to find the right UTF-8 'ç'.

Reply:
I swapped it to the normal c, if that is fine.